compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategiesGPU.cpp

```diff
@@ -266,13 +295,137 @@ void ReductionStrategyStagedThreadDistribution::compute(
     else
       vectorSizeStage2 = 1;
   }
+
+  // Stage 3
+  // -------
+  // Always happens by default with warp shuffles.
```

Comment:
This comment looks dangling. Maybe just say there's nothing to compute for stage 3.